tools/cbmc/proofs/make_common_makefile.py

```diff
@@ -195,16 +195,16 @@ def write_cbmcbatchyaml_target(opsys, _makefile):
 # Build configuration file to run cbmc under cbmc-batch in CI
 
 define encode_options
-'=$(shell echo $(1) | sed 's/ ,/ /g' | sed 's/ /;/g')='
+	'=$(shell echo $(1) | sed 's/ ,/ /g' | sed 's/ /;/g')='
```

Comment:
Why is this required for back porting? Other PR?

Reply:
Reverted to master.